src/server/handlers/consents.ts

```diff
@@ -48,18 +49,19 @@ export async function post (
   _context: Context,
   request: Request,
   h: ResponseToolkit): Promise<ResponseObject> {
+  // TODO: is this validation required? participantId removed from payload
   // Validate request
-  if (!isPostConsentRequestValid(request)) {
-    return h.response().code(Enum.Http.ReturnCodes.BADREQUEST.CODE)
-  }
+  // if (!isPostConsentRequestValid(request)) {
```

Comment:
Will this be done by the Hapi server? Maybe we need a test case to make sure (or maybe we already have one?)

Reply:
I will add comment that test is not required now, as we removed participant ID from body.
That validates whether request is valid by comparing if source header matches participant ID

Reply:
openAPI/swagger allows specifying per property constraints. More complex constraints spread over multiple properties/headers must be validated by custom code

Reply:
maybe we should drop `isPostConsentRequestValid` from domain/consents.ts and drop this commented code.

Reply:
Ok dropped that comment code